### DISCUSSION TITLE
i cant find rinkeby even after turning on test net on metamask, who else is having this issue?

### DISCUSSION THREAD

**Discussion by Scofield-Idehen** [smartcontractkit/full-blockchain-solidity-course-py#1825] [Q&A] [answered]
I can't find Rinkeby test network even after turning on the test net on Metamask. Who else is having this issue?

**Comment by cromewar** [comment, marked as answer]
Hello @Scofield-Idehen 
Since Ethereum changed from PoW to PoS the Rinkeby network was deprecated, please just use Goerli from now on.
Don't forget to check the code for each lesson on this repo as they were already updated to work with Goerli.

Finally by defualt Metamask does not show the testnets, so click on "show/hide networks" and activate the test networks.

**Comment by Scofield-Idehen** [comment]
I found a way to still use rinkeby, would update for anyone that still
wants to use it with metamask.

Thanks alot
